<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuPath">
    <property name="title">
     <string>Path</string>
    </property>
    <addaction name="actionClear_All"/>
    <addaction name="actionSend_All"/>
    <addaction name="actionUpdate_Astart"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuPath"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Alt+W</string>
   </property>
  </action>
  <action name="actionClear_All">
   <property name="text">
    <string>Clear All</string>
   </property>
  </action>
  <action name="actionSend_All">
   <property name="text">
    <string>Send All</string>
   </property>
  </action>
  <action name="actionUpdate_Astart">
   <property name="text">
    <string>Update Astart</string>
   </property>
   <property name="shortcut">
    <string>Alt+U</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
